keps/sig-scheduling/5004-dra-extended-resource/README.md

```diff
@@ -1077,10 +1151,14 @@ The Troubleshooting section currently serves the `Playbook` role. We may conside
 splitting it into a dedicated `Playbook` document (potentially with some monitoring
 details). For now, we leave it here.
 -->
+The troubleshooting section in https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/4381-dra-structured-parameters#troubleshooting
+still applies.
```

Comment:
Can you add something about the new resource claim creation adding to the kube apiserver load, and possibly tickling priority and fairness or whatever other rate controls api server has?

Reply:
done